Gemfile.lock

```diff
@@ -271,17 +271,17 @@ GEM
       railties (>= 5.0.0)
     faker (2.15.1)
       i18n (>= 1.6, < 2)
-    faraday (1.1.0)
+    faraday (1.2.0)
       multipart-post (>= 1.2, < 3)
       ruby2_keywords
     faraday_middleware (1.0.0)
       faraday (~> 1.0)
-    fastimage (2.1.1)
+    fastimage (2.2.0)
```

Comment:
dependency of `carrierwave-bombshelter` which we use to limit the image size of uploaded images, this new version adds a `FastImage.animated?` method